ua-snap/cmip6-utils
# Add fixed frequency variables to transfers pipeline

This PR revises the code in order to allow fixed frequency variables (anything under table ids `fx` and `Ofx`) to be transferred via Globus. This is one of the first steps in implementing the changes suggested in #48 

I had to explicitly adds some files found on MetaGrid under the `piControl` experiment that would not be captured under our normal audit. As a result, our holdings on ARDN now have an additional scenario directory (e.g. `/beegfs/CMIP6/arctic-cmip6/CMIP6/CMIP/MOHC/HadGEM3-GC31-MM/piControl/...`)

During this work, I re-ran the entire transfers pipeline which seemed to change quite a few of the batch files. This might be because file availability changed on Globus. 


**TO TEST:**

Run the holdings audit (via `esgf_holdings.py` and `esgf_holdings_e3sm.py`) and generate the batch files and manifest (via `generate_batch_files.py` and `generate_manifest.py`) and inspect the CSVs and batch files to make sure nothing has been modified. It is probably not necessary to run the entire transfer again, but I'll leave that up to you. 


*** Unrelated to the rest of this PR, I noticed some missing modiles in `utils.py` and there are some functions in there that I think are unused. Made an issue about this, I may refactor at some point and clean it up.
